jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -66,6 +66,9 @@ def parse_data(app, file_obj, data_type=None, data_label=None, parent=None):
                        flux_viewer_reference_name=flux_viewer_reference_name)
             return
 
+        # try parsing file_obj as a URI/URL:
+        file_obj = download_uri_to_path(file_obj, cache=cache)
```

Comment:
This seems to not be handling the case of the user passing the path to a local file on disk correctly:

```
File [~/projects/jdaviz/jdaviz/utils.py:452](http://localhost:8888/lab/tree/~/projects/jdaviz/jdaviz/utils.py#line=451), in download_uri_to_path(possible_uri, cache, local_path)
    449     return download_file(possible_uri, cache=cache)
    451 else:
--> 452     raise ValueError(f"URI {possible_uri} with scheme {parsed_uri.scheme} is not "
    453                      f"currently supported.")
    455 # assume this isn't a URI after all:
    456 return possible_uri

ValueError: URI [/User/username/Data/test_cubes/jw02732-o004_t004_miri_ch1-shortmediumlong_s3d.fits](http://localhost:8888/User/username/Data/test_cubes/jw02732-o004_t004_miri_ch1-shortmediumlong_s3d.fits) with scheme  is not currently supported.
```

Reply:
It looks like the string that was passed is a markdown link – what were your inputs?

Reply:
I don't know where the markdown came from, but as we discussed offline this does work if I actually get the path to my file correct, I think we just need a slightly more informative error for the case where the filepath provided doesn't exist.

Reply:
We solved this offline – the file path did not exist locally, but the error message wasn't informative. I put a more informative error message in fa83f48.